dev-vcs/git-pw/git-pw-2.0.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..9} )
+DISTUTILS_USE_SETUPTOOLS=bdepend
+
+inherit distutils-r1
+if [[ "${PV}" == 99999 ]] ; then
```

Comment:
I don't think it's standardized, but I generally see (and use) 4 `9`'s.

Reply:
you right, my typo